plugins/techdocs-backend/src/DocsBuilder/builder.ts

```diff
@@ -192,11 +197,16 @@ export class DocsBuilder {
       )}`,
     );
 
-    await this.publisher.publish({
+    const published = await this.publisher.publish({
       entity: this.entity,
       directory: outputDir,
     });
 
+    // Invalidate the cache for any published objects.
+    if (this.cache && published && published?.objects?.length) {
```

Comment:
_Nit_: When `grep`-ing through the source I can see we have roughly an equal amount of implicit non-zero length checking and explicit. Do we have a convention whether to use `if (length)` vs `if (length > 0)`?